Microsoft.Dynamics365.UIAutomation.Api/Pages/Navigation.cs

```diff
@@ -468,5 +468,27 @@ public BrowserCommandResult<bool> SignOut(int thinkTime = Constants.DefaultThink
                 return true;
             });
         }
+        #region Function to Switch to a Window based on the index
+        public BrowserCommandResult<bool> SwitchToAnotherWindow(int index)
+        {
+            return this.Execute(GetOptions("switch handles"), driver =>
+            {
+                System.Collections.ObjectModel.ReadOnlyCollection<string> windowHandles = driver.WindowHandles;
+                string popupHandle = string.Empty;
+
+
+                foreach (string handle in windowHandles)
+                {
+                    if (handle != driver.CurrentWindowHandle)
```

Comment:
Are you wanting to switch to the last window or whichever window is not the Current window.  What if you have multiple windows?